### PR TITLE
Windup 3997: mtr 1 2 0 release notes hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
Resolves: https://issues.redhat.com/browse/WINDUP-3997

MTR 1.2.0

Preview: https://deploy-preview-736--windup-documentation.netlify.app/docs/release-notes-mtr/master/

### CODE REVIEW

File: docs/topics/mtr-rn-new-features-1-2-0.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-1-2-0_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.2.0.
+
+. Decompile and analyze Java 17 based applications
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-new-features-1-2-0.adoc:11]: @HagayVider1 
I think this bullet should be a noun(s) for the sake of uniformity, because all other bullets are (enhancements, compatibility, upgrade). I would suggest "Decompilation and analysis of applications based on Java 17".

File: docs/topics/mtr-rn-new-features-1-2-0.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+. Rules Override enhancement: New degree of selectivity that only overrides a rule if `rulesetId` and `ruleId` match, and the target of the override ruleset matches the target provided for the analysis
+. Eclipse Plugin Java 17 compatibility
+. Upgrade of the Windup Operator: Adopted `Quarkus 2.13.7.Final` and the `Quarkus Operator SDK 4.0.8`
+
+[id="rn-new-rulesets-targets-1-2-0"]
+== New rulesets and targets
+
+. OpenJDK 21: Rules to support the upgrading to OpenJDK 21.
+. Red Hat JBoss Web Server 6: Rules to support the upgrade of JWS / Tomcat applications to JWS 6 / Tomcat 10.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-new-features-1-2-0.adoc:20]: @HagayVider1 
Slashes are frowned upon by the IBM style guide. I would suggest changing to "...JWS and Tomcat applications to JWS 6 and Tomcat 10."

File: docs/topics/mtr-rn-new-features-1-2-0.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+. Camel 4: Comprehensive rulesets supporting upgrade to all Y stream releases of Camel 3 and Camel 4.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-new-features-1-2-0.adoc:21]: @HagayVider1 
"...all Y-stream releases..."

File: docs/topics/mtr-rn-new-features-1-2-0.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+. More migration rules to support Red Hat JBoss EAP 8 and Hibernate 6.
+// Edited the language. Please review if acceptable.
+. Java/Jakarta EE to Quarkus: New rulesets support migrating Java/Jakarta EE applications to Quarkus 3. These rulesets cover the _quarkifcation_ of the project, along with JAX-RS and CDI technologies. Additional rules that support this migration path are still under development and will be made available in future Z stream releases.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-new-features-1-2-0.adoc:24]: @HagayVider1 
I don't know why Phil wants to use this strange word, but it seems to need another 'i' - _quarkification_.
Also, "...Z-stream...".

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 5]
```diff
@@ -9,12 +9,12 @@
 
 This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
 
-.Integration with JIRA
+.xref:../../../docs/web-console-guide/master.adoc#creating-configuring-jira-connection[Integration with JIRA]
```
Comment (sshveta) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:12]: Why are we editing MTA new features in MTR release notes PR . These changes are un related . 
Can we put them in separate PR?

**Review by stillalearner — APPROVED** [review]
lgtm
